Solutions/CrowdStrike Falcon Endpoint Protection/Package/createUiDefinition.json

```diff
@@ -0,0 +1,442 @@
+                }
+              },
+              {
+                "name": "playbook4-Playbook_Name",
+                "type": "Microsoft.Common.TextBox",
+                "label": "Playbook Name",
+                "defaultValue": "CrowdStrike_ResponsefromTeams",
+                "toolTip": "Please enter Playbook Name",
+                "constraints": {
+                  "required": true,
+                  "regex": "[a-z0-9A-Z]{1,256}$",
+                  "validationMessage": "Please enter the Playbook Name"
+                }
+              },
+              {
+                "name": "playbook4-Teams_GroupId",
+                "type": "Microsoft.Common.TextBox",
+                "label": "Playbook Name",
+                "defaultValue": "TeamgroupId",
+                "toolTip": "Please enter Teams Group Id",
+                "constraints": {
+                  "required": true,
+                  "regex": "[a-z0-9A-Z]{1,256}$",
+                  "validationMessage": "Please enter Teams Group Id"
+                }
+              },
+              {
+                "name": "playbook4-Teams_ChannelId",
+                "type": "Microsoft.Common.TextBox",
+                "label": "Playbook Name",
+                "defaultValue": "TeamChannelId",
+                "toolTip": "Please enter Teams Channel Id",
+                "constraints": {
+                  "required": true,
+                  "regex": "[a-z0-9A-Z]{1,256}$",
+                  "validationMessage": "Please enter Teams Channel Id"
+                }
+              }
+            ]
+          },
+          {
+            "name": "playbook5",
+            "type": "Microsoft.Common.Section",
+            "label": "Crowdstrike_Remediation_Host",
+            "elements": [
+              {
+                "name": "playbook5-text",
+                "type": "Microsoft.Common.TextBlock",
+                "options": {
+                  "text": "This playbook will Contain the device automatically without SOC intervention"
+                }
+              },
+              {
+                "name": "playbook5-Playbook_Name",
+                "type": "Microsoft.Common.TextBox",
+                "label": "Playbook Name",
+                "defaultValue": "Crowdstrike_Remediation_Host",
+                "toolTip": "Please enter Playbook Name",
+                "constraints": {
+                  "required": true,
+                  "regex": "[a-z0-9A-Z]{1,256}$",
+                  "validationMessage": "Please enter the Playbook Name"
+                }
+              }
+            ]
+          }
+        ]
+      }
+    ],
+    "outputs": {
+      "workspace-location":"[first(map(filter(basics('getLAWorkspace').value, (filter) => and(contains(filter.id, toLower(resourceGroup().name)),equals(filter.name,basics('workspace')))), (item) => item.location))]",
```

Comment:
This 'workspace-location' value is the main change made here. This function gets the location of the workspace, rather than the resource group.